src/JupyterLibrary/clients/jupyterlab/Selectors.resource

```diff
@@ -29,7 +29,7 @@ ${JLAB XP CARD}                         //div[contains(@class, 'jp-LauncherCard'
 ${JLAB XP DOCK}                         //div[@id='jp-main-dock-panel']
 ${JLAB XP MENU ITEM}                    //li[contains(@class, 'p-Menu-item')]
 ${JLAB XP MENU ITEM LABEL}              //div[contains(@class, 'p-Menu-itemLabel')]
-${JLAB XP MENU ITEM ENABLED}            ${JLAB XP MENU ITEM}\[not(contains(@class,'p-mod-disabled'))]
+${JLAB XP MENU ITEM ENABLED}            ${JLAB XP MENU ITEM}\[not(contains(@class,'p-mod-hidden'))]
```

Comment:
In light of this change, let's update the variable name from `ENABLED` to `VISIBLE` here (and at the call site).

Is there a way we can emulate and test the failing condition? E.g. some environment variable or launch technique where it will _think_ it's in Hub enough to show these?

Reply:
Updated in the latest commit.
As for your question, I am not sure if there is any way to do that short of installing and launching JupyterHub itself. I can test it against my own environment, but if your aim was to have an automated test in this repo's CI that wouldn't really solve it.

Reply:
> installing and launching JupyterHub itself. 

I'll hoist to a separate issue: i feel like there must be a way to set up some dummy config values or environment variables to make it do the thing.